### PR TITLE
Fixes according Comments

### PR DESCRIPTION
**Pull request by MartinYDimitrov**

### CODE REVIEW

File: src/common/variables.scss [hunk 1]
```diff
@@ -4,7 +4,8 @@ $secondary: rgba(128, 128, 128, 0.5);
 $tertiary: rgba(255, 165, 0, 0.5);
 
 //pixels
-$primarypixels: 100px;
+$primaryHeight: 6.25rem;
+$doubleHeight: 2 * $primaryHeight;
```
Comment (ximet) [inline src/common/variables.scss]: doubleHeight for WHAT? The article, a navigation panel, skyscraper?

File: src/common/variables.scss [hunk 2]
```diff
@@ -4,7 +4,8 @@ $secondary: rgba(128, 128, 128, 0.5);
...
 
 //percentage
```
Comment (ximet) [inline src/common/variables.scss]: percentage, but you're using REM

File: src/common/variables.scss [hunk 3]
```diff
@@ -4,7 +4,8 @@ $secondary: rgba(128, 128, 128, 0.5);
...
-$primarypercent: 15%;
+$primaryWidth: 12rem;
```
Comment (ximet) [inline src/common/variables.scss]: I don't like naming primary, secondary, etc.. It's not obvious

File: src/components/FilterableList/FilterableList.scss [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
+  top: $primaryHeight;
   left: 0;
   padding: 0 10px;
   background-color: $primary;
   height: auto;
-  width: 15%;
+  width: $primaryWidth;
   overflow-y: scroll;
-  max-height: calc(100vh - 200px);
+  max-height: calc(100vh - #{$doubleHeight});
```
Comment (ximet) [inline src/components/FilterableList/FilterableList.scss]: I'm sure, you can avoid calc methods in your CSS

File: src/PageLayout/PageLayout.jsx [hunk 5]
```diff
@@ -12,4 +13,8 @@ function PageLayout({ children }) {
   );
 }
 
+PageLayout.propTypes = {
+  children: PropTypes.element
```
Comment (AV29) [inline src/PageLayout/PageLayout.jsx]: Better go with PropTypes.node - it's gonna work for children of any kind... strings, numbers etc... anything that could be rendered

File: src/components/Button/Button.jsx [hunk 6]
```diff
@@ -1,13 +1,17 @@
...
   return (
-    <button type="button" className={styles['menu-btn']} onClick={clickHandler}>
+    <button type="button" className={styles.menuBtn} onClick={onClick}>
       Menu
     </button>
   );
 }
 
+Button.propTypes = {
+  onClick: PropTypes.func
```
Comment (AV29) [inline src/components/Button/Button.jsx]: what will happen if the consumer of your Button doesn't pass onClick handler ?

hint: the code will fail. You gotta define .Required prop or a default prop for onClick

File: src/components/Footer/Footer.jsx [hunk 7]
```diff
@@ -4,13 +4,17 @@ import styles from './Footer.scss';
...
 
-  const timerID = setInterval(() => setDate(new Date()), 1000);
+  useEffect(() => {
+    const timerId = setInterval(() => setDate(new Date()), 1000);
 
-  useEffect(() => {}, [timerID]);
+    return () => {
+      clearTimeout(timerId);
+    };
+  }, [date]);
```
Comment (AV29) [inline src/components/Footer/Footer.jsx]: In this case you don't have to specify date in a list of dependencies. Try it and make sure you understand why exactly

File: src/common/variables.scss [hunk 8]
```diff
@@ -3,8 +3,9 @@ $primary: rgba(173, 216, 230, 0.5);
 $secondary: rgba(128, 128, 128, 0.5);
 $tertiary: rgba(255, 165, 0, 0.5);
 
-//pixels
-$primarypixels: 100px;
+//height
+$defaultHeight: 6.25em;
```
Comment (ximet) [inline src/common/variables.scss]: default for what? for some div, or nav panel or etc?

File: src/components/Button/Button.scss [hunk 9]
```diff
@@ -1,8 +1,8 @@
 @import '../../common/variables.scss';
 
-.menu-btn {
-  width: 60%;
+.menuBtn {
```
Comment (ximet) [inline src/components/Button/Button.scss]: .menuButton

**Review by AV29 — APPROVED** [review]

**Review by ximet — APPROVED** [review]

**Pull request merged**